pkg/restapi/rp/operation/operations.go

```diff
@@ -152,6 +156,10 @@ type vc struct {
 	FlowType string `json:"flowType"`
 }
 
+type initiate struct {
```

Comment:
Typically struct types should be nouns instead of verbs.

(I realize `initiate` is also a noun, but I don't think the noun form makes sense here as it generally refers to a person)

Reply:
Do you need this `initiate` struct, or can you use a `[]profileView` directly?

Reply:
As discussed one-on-one, renamed this to reflect the purpose of the `struct`, but refrained from providing generic data object to the HTML template for clarity purposes.